From 50e45158994a1cd963982a1bfb25e809d498d511 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 30 Aug 2017 11:59:03 +0800
Subject: [PATCH 2467/5242] MLK-16301-2 gpu: imx: dpu: common: Remove the list
 in dpu plane group

commit  13a55ac6d4fd69da3646a2fbb7cd4f1fc5440491 from
https://source.codeaurora.org/external/imx/linux-imx.git

No one is using the list in the dpu plane group, so let's remove it and
the mutex lock which protects the list.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-common.c |    2 --
 include/video/dpu.h              |    2 --
 2 files changed, 4 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-common.c b/drivers/gpu/imx/dpu/dpu-common.c
index 15f14ef..30aff1d 100644
--- a/drivers/gpu/imx/dpu/dpu-common.c
+++ b/drivers/gpu/imx/dpu/dpu-common.c
@@ -970,8 +970,6 @@ static int dpu_add_client_devices(struct dpu_soc *dpu)
 	else
 		memcpy(reg, &client_reg[2], reg_size);
 
-	INIT_LIST_HEAD(&plane_grp->list);
-	mutex_init(&plane_grp->lock);
 	plane_grp->id = id / client_num;
 	plane_grp->has_vproc = display_plane_video_proc;
 
diff --git a/include/video/dpu.h b/include/video/dpu.h
index bd2582e..e208ced 100644
--- a/include/video/dpu.h
+++ b/include/video/dpu.h
@@ -671,8 +671,6 @@ struct dpu_plane_res {
  */
 struct dpu_plane_grp {
 	struct dpu_plane_res	res;
-	struct list_head	list;
-	struct mutex		lock;
 	unsigned int		hw_plane_num;
 	unsigned int		hw_plane_fetcheco_num;
 	unsigned int		hw_plane_hscaler_num;
-- 
1.7.9.5

